[2024-12-13T13:14:50.236+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shift_table_dag.shift_table_dag manual__2024-12-13T13:14:49.008443+00:00 [queued]>
[2024-12-13T13:14:50.243+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shift_table_dag.shift_table_dag manual__2024-12-13T13:14:49.008443+00:00 [queued]>
[2024-12-13T13:14:50.243+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-13T13:14:50.333+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): shift_table_dag> on 2024-12-13 13:14:49.008443+00:00
[2024-12-13T13:14:50.338+0000] {standard_task_runner.py:57} INFO - Started process 3202 to run task
[2024-12-13T13:14:50.340+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shift_table_dag', 'shift_table_dag', 'manual__2024-12-13T13:14:49.008443+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpo7g8q8yf']
[2024-12-13T13:14:50.343+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask shift_table_dag
[2024-12-13T13:14:50.360+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-13T13:14:50.396+0000] {task_command.py:415} INFO - Running <TaskInstance: shift_table_dag.shift_table_dag manual__2024-12-13T13:14:49.008443+00:00 [running]> on host 0406bded5f12
[2024-12-13T13:14:50.474+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shift_table_dag' AIRFLOW_CTX_TASK_ID='shift_table_dag' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T13:14:49.008443+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T13:14:49.008443+00:00'
[2024-12-13T13:14:50.675+0000] {base.py:73} INFO - Using connection ID 'admin' for task execution.
[2024-12-13T13:14:50.752+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-13T13:14:50.762+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=shift_table_dag, task_id=shift_table_dag, execution_date=20241213T131449, start_date=20241213T131450, end_date=20241213T131450
[2024-12-13T13:14:50.794+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-13T13:14:50.811+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
